pkgs/applications/misc/slic3r/default.nix

```diff
@@ -1,22 +1,25 @@
 { stdenv, fetchgit, perl, makeWrapper, makeDesktopItem
-, which, perlPackages
+, which, perlPackages, fetchFromGitHub, boost, linuxHeaders
 }:
 
 stdenv.mkDerivation rec {
-  version = "1.2.9";
+  version = "1.3.0-dev";
```

Comment:
IMHO nixpkgs should package the latest _releases_ from upstream. Version 1.3.0-dev is not a release.

Reply:
That is true.  I asked upstream, and was told that a current version is recommended over release 1.2.9 for production use